internal/impl/redis/processor.go

```diff
@@ -36,8 +36,7 @@ performed for each message and the message contents are replaced with the result
 			Description("A [Bloblang mapping](/docs/guides/bloblang/about) which should evaluate to an array of values matching in size to the number of arguments required for the specified Redis command.").
 			Version("4.3.0").
 			Example("root = [ this.key ]").
-			Example(`root = [ meta("kafka_key"), this.count ]`).
-			Default(``)).
+			Example(`root = [ meta("kafka_key"), this.count ]`)).
```

Comment:
We need to set this as optional unfortunately, otherwise the backwards compatible `operator` field isn't possible anymore. We could potentially compromise by setting to optional but adding a linting rule specifically that checks either `args_mapping` or `operator` is present.